### ISSUE TITLE
a11y: Review Input Accessibility

### ISSUE THREAD

**Issue by StephenRiosDev** [AlaskaAirlines/auro-formkit#454]
### Description

Assess accessibility of the Input component. Ensure it meets all W3C standards and that the screen reader functions as expected.

Input should:

- Behave in the same manner as a normal HTMLInputElement DOM element
- Errors should be read to the user when they are generated

### Known Issues

1. Inputs currently have `aria-invalid="true"` attribute by default on page load.

### Exit criteria (To be determined by the Auro team)

Input component is accessible to non-sighted users and meets or exceeds all other W3C WCAG 2.0 AA criteria.

**Comment by blackfalcon** [comment]
The floor-wide standard is to meet or exceed WCAG 2.2 Level AA. Please see [Alaska Airlines Digital Accessibility and Inclusion Policy](https://wiki.devtools.teamaag.com/standards/accessibility/digital-accessibility-policy) for more information.
